### ISSUE TITLE
Empty iframe src causes an error when auto-filling credentials

### ISSUE THREAD

**Issue by ampldavorp** [passbolt/passbolt_browser_extension#172]
## Empty iframe src causes an error when auto-filling credentials
* Passbolt Version: 3.11.2
* Platform and Target:
-- Browser: Chrome 110.0.5481.178 (Official Build) (64-bit)
-- Windows 10 Pro (21H2)

### What you did
I have the following case:
```
<iframe src="about:blank"></iframe>
<iframe src="login.html"></iframe>
```

### What happened
When I try to auto-fill credentials, the following error message appears in Passbolt: **Unable to use the password on this page. Copy and paste the information instead.**
There is also an error in the browser console: **TypeError: Failed to construct 'URL': Invalid URL**
Credentials are not auto-filled.

![image](https://user-images.githubusercontent.com/127186040/223367908-5b7fc114-7c0b-470f-8591-20c8544af779.png)

### What you expected to happen
Credentials should be successfully auto-filled.

If I move the iframe with the login form before the empty iframe, credentials will be auto-filled successfully. But I cannot do that in all cases.

![image](https://user-images.githubusercontent.com/127186040/223368021-92a340e5-ea86-4c03-9362-6af0d0a21026.png)

**Comment by stripthis** [comment]
Thank you for the report @ampldavorp . We'll schedule some time to look into it.
Ref. PB-23547

**Comment by ampldavorp** [comment]
Thank you @stripthis.

We are both community and business edition user - an email was also sent to your tech support.

**Comment by Benj1er** [comment]
Fixed with v4.0.0, don't hesitate to reopen it if the issue persists.
